js/scripts.js

```diff
@@ -0,0 +1,5 @@
+var withButtonClass = document.getElementsByClassName('btn');
+console.log(withButtonClass);
+for (i=0; i<withButtonClass.length; i++){
```

Comment:
declare variable `i` and clean your code, add spaces

move `withButtonClass.length` to new variable